dd-sdk-android/src/main/kotlin/com/datadog/android/log/Logger.kt

```diff
@@ -248,20 +225,24 @@ private constructor(
             return this
         }
 
-        private fun overrideLogStrategy(strategy: LogStrategy): Builder {
-            logStrategy = strategy
-            return this
-        }
+        // region Internal
 
-        private fun overrideNetworkInfoProvider(provider: NetworkInfoProvider): Builder {
-            networkInfoProvider = provider
-            return this
+        private fun buildLogcatLogger(): LogcatLogHandler {
+            return LogcatLogHandler(serviceName)
         }
 
-        private fun overrideTimeProvider(provider: TimeProvider): Builder {
-            timeProvider = provider
-            return this
+        private fun buildDatadogLogger(): DatadogLogHandler {
```

Comment:
it is confusing this function name. We should name it _buildDatadogLoggerHandler_

Reply:
Done